packages/modules/neo4j/src/neo4j-container.test.ts

```diff
@@ -83,4 +83,34 @@ describe("Neo4jContainer", { timeout: 180_000 }, () => {
     await container.stop();
   });
   // }
+
+  // plugins list {
```

Comment:
Could you please add this example to the module docs?

Reply:
Whoops, forgot these had to be manually added. Added to docs 👍